src/test/java/school/hei/haapi/service/SesServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package school.hei.haapi.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import school.hei.haapi.service.aws.SesService;
+import software.amazon.awssdk.services.ses.SesClient;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+
+class SesServiceTest {
+  private final static String sender = "contact@hei.school";
```

Comment:
No. 

Set them as local variable inside the `student_get_email_response_ok()` instead of this.